0015/README.md

```diff
@@ -0,0 +1,147 @@
+---
+id: 0015
+title: HTTP Routing For Services
+status: ideation
+authors: Manny <manny@packet.com>
+---
+
+## Summary
+
+HTTP routing for services started by tink-worker running on "provisioner" machines.
+
+## Goals and not Goals
+
+#### Goal
+
+A way for provisonee/target machines to make HTTP(S) requests to a service that is defined as part of its workflow.
```

Comment:
In Tinkerbell we call them `hardware`. let's reduce the number of terms we use when possible.

Reply:
Well yes, but I want to separate different kinds of hardware/workers. Is hardware always meant to be the machine that is being installed/most-of-the-workflow-is-actually-for?

Reply:
That's the way I understood that, but I am open to learn more about what hardware means.

Reply:
Yeah we don't have any term internally for these things, its just the "site prov" machines or the actual host names, but nothing generic. :thinking: we should maybe figure one out.

Reply:
oh I think I've heard "remote-worker" vs "local-worker" earlier on in the design/beginnings of tink. That might be good to use here?